src/internal/storage/chunk/gc_test.go

```diff
@@ -2,7 +2,7 @@ package chunk
 
 import (
 	"context"
-	"io"
+	io "io"
```

Comment:
is this a weird editor/goimports thing or is there some collision this is avoiding?

Reply:
editor / goimport, I don't know why that happens.